stdlib/public/Cxx/std/String.swift

```diff
@@ -186,10 +182,34 @@ extension String {
   /// - Complexity: O(*n*), where *n* is the number of bytes in the C++ UTF-16
   ///   string.
   public init(_ cxxU16String: std.u16string) {
+    self = cxxU16String.withUTF16 {
+      String(decoding: $0, as: UTF16.self)
+    }
+  }
+}
+
+// MARK: Converting a C++ string to a C string
+
+extension std.string {
+  @inlinable
+  public __consuming func withUTF8<Result>(
```

Comment:
it's not a consuming function. You need `borrowing`.

Reply:
Oops, tried comparing them and pushed the wrong thing